twopence/run

```diff
@@ -366,7 +366,7 @@ def nfs_write_exports(server, exports):
 		journal.failure("unable to upload exports file")
 		return False
 
-	if nfs_server_control(server, "reload"):
+	if nfs_server_control(server, "restart"):
```

Comment:
I need to test this change, with the change line 221, reload command was failing,(before the 2 changes) now we get locally and in cloud  a failure here : 
```
---------------------------------
TEST: export read-only to 192.168.0.0/16
server: updating exports file:
/srv/nfs/dir1   192.168.0.0/16(ro,no_subtree_check) *(rw,no_root_squash,no_subtree_check)
---end of exports file---
server: uploading /etc/exports
server: systemctl restart nfsserver.service
Flush kernel export cache
server: exportfs -f
server: wc -l < /proc/fs/nfsd/exports
2
Trying to get list of exported file systems via exportfs
server: exportfs -v
Exported file systems (as seen by mountd):
/srv/nfs/dir1 	192.168.0.0/16(ro,wdelay,root_squash,no_subtree_check,sec=sys,ro,secure,root_squash,no_all_squash)
/srv/nfs/dir1 	<world>(rw,wdelay,no_root_squash,no_subtree_check,sec=sys,rw,secure,no_root_squash,no_all_squash)
---end of exportfs output---
Trying to retrieve kernel export table
server: cat /proc/fs/nfsd/exports
Content of kernel export table:
# Version 1.1
# Path Client(Flags) # IPs
---end of kernel exports data---
verify that dir1 is exported read-only (mounted with options vers=4,soft,intr,soft,intr)
server: /bin/rm -f /srv/nfs/dir1/readonly-test
client1: mounting /srv/nfs/dir1 from 192.168.15.239 (options vers=4,soft,intr,soft,intr)
client1: /bin/mount 192.168.15.239:/srv/nfs/dir1 /mnt/dir1 -o vers=4,soft,intr,soft,intr
client1: /bin/touch /mnt/dir1/readonly-test
Testsuite error: client1: command execution failed with exception
ERROR: client1: command execution failed with exception
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/susetest.py", line 247, in run
    status = super(Target, self).run(cmd)
SystemError: run: Remote command took too long to execute

```